### PR TITLE
fix: output of path invalid value in the case of null value

### PR DESCRIPTION
**Pull request by pjm17971**
Previously when calculating rates if either the previous or current values were null pond would emit a console.log message saying the `field value is invalid or missing`. In the case of nulls though it is better to just output a null rate and not pollute logs. Nulls are often seen in timeseries when incomplete data is present.

### CODE REVIEW

**Comment by jdugan1024** [comment]
Looks like the travis tests failed due to an incompatible version of node.

File: packages/pond/src/rate.ts [hunk 1]
```diff
@@ -41,81 +41,87 @@ import { RateOptions } from "./types";
...
+      ) {
+        // Only issue warning if the current or previous values are bad
+        // i.e. not a number or not null (null values result in null output)
+        console.warn(
+          `Event field "${fieldPath}" is a non-numeric or non-null value`
+        );
+      }
+
+      d =
+        this.allowNegative === false && rate < 0
```
Comment (jdugan1024) [inline packages/pond/src/rate.ts]: Is using the ternary operator here a prettier thing?  I find reading this slightly jarring, but not the end of the world.  Just wondering if it's you or prettier preferring this form...

Comment (pjm17971) [reply packages/pond/src/rate.ts]: tslint prefers it, so I went with it.

**Comment by pjm17971** [comment]
Travis is now fixed, so merging.

**Pull request merged**